front_file_hosting/src/App.vue

```diff
@@ -1,30 +1,88 @@
 <template>
   <div id="nav">
     <router-link to="/">Home</router-link> |
-    <router-link to="/about">About</router-link>
+    <router-link to="/register">Sign up</router-link> |
+    <router-link to="/login">log in</router-link> |
+    <router-link to="/user">User</router-link> |
+    <router-link to="/user/change_password">Change Password</router-link> |
+    <router-link to="/files">Files</router-link> |
+    <router-link to="/upload">Upload</router-link> |
+    <router-link to="/file_detail">File</router-link> |
   </div>
+
+  <div class="container d-flex flex-column flex-md-row align-items-center pb-3 mb-4 border-bottom">
+    <a href="/" class="d-flex align-items-center text-dark text-decoration-none">
+        <span class="fs-4"><i>The best storage for your files</i></span>
+    </a>
+    <nav class="d-inline-flex mt-2 mt-md-0 ms-md-auto">
+        <a class="me-3 py-2 text-dark text-decoration-none" href="#/">Home</a>
+        <a class="me-3 py-2 text-dark text-decoration-none" href="#/register">Sign Up</a>
+        <a class="py-2 me-4 text-dark text-decoration-none" href="#/login">Login</a>
+        <a class="py-2 me-4 text-dark text-decoration-none" href="#/user">Profile</a>
+        <a class="py-2 me-4 text-dark text-decoration-none" href="#/files">Files</a>
+        <a class="btn btn-primary" href="http://127.0.0.1:1338/api/files/template/">Upload File</a>
```

Comment:
I don't think that's a good idea to have the "magic" URL, but I don't know how to properly deal with URLs in Vue, so at least, I think you should create `constants` or `config` file, so would be easier to update hostname, port and that's kind of stuff if anything changes.